testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/vault/KeycloakVaultTest.java

```diff
@@ -37,7 +37,7 @@
  *
  * @author <a href="mailto:sguilhen@redhat.com">Stefan Guilhen</a>
  */
-@EnableVault
+@EnableVault(providerId = EnableVault.PROVIDER_ID.PLAINTEXT)
```

Comment:
I think we should rather make the `KeycloakVaultTest` abstract and create two derived classes – `KeycloakPlaintextVaultTest` and `KeycloakKeystoreVaultTest` (we already have that one).

Reply:
I simplified and changed the structure in order to improve readability. So right now we have the parent class `KeycloakVaultTest` with the common functionality for both `KeycloakKeystoreVaultTest` and `KeycloakKeystorePlaintextTest`. These two just start the server with necessary parameters while using a different provider (based on the `@EnableVault` annotation). All assertion is happening the parent class now.

I don't think that an abstraction is necessary - right now there is no need for overriding a method with specific functionality, the code is the same for both the children. WDYT?